graphql-spring-boot-autoconfigure/build.gradle

```diff
@@ -23,6 +23,8 @@ dependencies {
 
     compile "org.springframework.boot:spring-boot-autoconfigure:$LIB_SPRING_BOOT_VER"
     compile "org.springframework.boot:spring-boot-starter-websocket:$LIB_SPRING_BOOT_VER"
+    compile "javax.transaction:javax.transaction-api:$LIB_TRANSACTIONS_API_VERSION"
```

Comment:
Can these be included as `compileOnly` instead so we're not pulling in more dependencies? People that want to use this will probably have these dependencies themselves anyway already, won't they?

Reply:
Sure, made them ```compileOnly```